docs/projects/ad7606x_fmc/index.rst

```diff
@@ -0,0 +1,438 @@
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`AD7606` is a 8-/6-/4-Channel DAS with 16-Bit, Bipolar Input, 
+Simultaneous Sampling ADC. Each part contains analog input clamp protection, a
+second-order antialiasing filter, a track-and-hold amplifier, a 16-bit charge
+redistribution successive approximation analog-to-digital converter (ADC), a 
+flexible digital filter, a 2.5 V reference and reference buffer, and high speed
+serial and parallel interfaces. The :adi:`AD7606` operate from a single 5 V 
+supply and can accommodate ±10 V and ±5 V true bipolar input signals while
+sampling at throughput rates up to 200 kSPS for all channels. The input clamp
+protection circuitry can tolerate voltages up to ±16.5 V. The AD7606 has 1 MΩ
+analog input impedance regardless of sampling frequency. The single supply
+operation, on-chip filtering, and high input impedance eliminate the need for
+driver op amps and external bipolar supplies.
+
+The :adi:`AD7606C` is a directly pin replacement (software and hardware) for
+both AD7608 and AD7609, with higher input impedance, throughput rate and 
+extended temperature range with additional features such as 16/18-bit sample
+size, system gain/offset/phase calibration, sensor disconnect detection, 
+lower Vdrive operation, diagnostics, additional oversampling ratios and per
+channel analog input range selection with bipolar differential, bipolar
+single-ended and unipolar single-ended options.
+
+The :adi:`EVAL-AD7606B-FMCZ <EVAL-AD7606B-FMCZ>` and 
+:adi:`EVAL-AD7606C-18 <EVAL-AD7606C-18>` evaluation boards are designed to 
+help users to easily evaluate the features of :adi:`AD7606B`, 
+:adi:`AD7606C-16` and :adi:`AD7606C-18` analog-to-digital converters (ADCs).
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD7606B <EVAL-AD7606B>`
+-  :adi:`EVAL-AD7606C-16 <AD7606C-16>`
+-  :adi:`EVAL-AD7606C-18 <EVAL-AD7606C-18>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD7606B`
+-  :adi:`AD7606C-16`
+-  :adi:`AD7606C-18`
+-  :adi:`ADP7118`
+-  :adi:`ADR4525`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC slot
+
+Block design
+-------------------------------------------------------------------------------
+
+The data path of the HDL design is simple as follows:
+
+-  the parallel interface is controlled by the
+   :dokuwiki:`axi_ad7606x <resources/fpga/docs/axi_ad7606x>` IP core
+-  the serial interface is controlled by the :ref:`SPI_Engine <spi_engine>`
+   Framework
+-  data is written into memory by a DMA (:ref:`axi_dmac core <axi_dmac>`)
+-  all the control pins of the device are driven by GPIOs
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Please add this phrase: https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/ad7616_sdz/index.rst?plain=1#L60

Reply:
Done.